homeassistant/components/binary_sensor/ping.py

```diff
@@ -126,7 +129,18 @@ def ping(self):
                     'avg': rtt_avg,
                     'max': rtt_max,
                     'mdev': ''}
-
+            if 'max/' not in str(out):
+                _LOGGER.debug("Detected Alpine ping...")
```

Comment:
This aids for when the ones who don't care complain it doesn't work and send a log with debug in it so we can work out what is going on. I'll just cut it cause I want to get it working.